### ISSUE TITLE
rspamd_monitored_propagate_error

### ISSUE THREAD

**Issue by damned-trooper** [vstakhov/rspamd#962]
Hi, after update to version 1.3.4 from git, i've got a lot of errors in /var/log/messages:
```
Sep 19 19:44:07 mx01 rspamd[11085]: <khr3t6>; monitored; rspamd_monitored_propagate_error: timeout on resolving multi.surbl.org, 1 retries left
Sep 19 19:44:07 mx01 named[11543]: error (unexpected RCODE NOTIMP) resolving 'facebook.sbl.spamhaus.org/TYPE43744/IN': 139.59.48.139#53
Sep 19 19:44:08 mx01 named[11543]: error (unexpected RCODE NOTIMP) resolving 'facebook.sbl.spamhaus.org/TYPE43744/IN': 194.104.0.140#53
```
On servers with old version of rspamd there are no erros like this.

How can i troubleshoot this?

**Comment by moisseev** [comment]
Fixed in `master`: 0296d8f657421cd05b021df856d7aa2f3cbac00d, fda9cd3e09880af4b437830010bef6044c5e1436

**Comment by damned-trooper** [comment]
Thanks a lot.
